### PR TITLE
Add managed attribute to instance cluster

### PR DESCRIPTION
**Pull request by sfang97**
Add managed attribute to instance cluster

### CODE REVIEW

**Review by svanharmelen — APPROVED** [review]
Thanks @sfang97 👍

File: instance_clusters.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ type InstanceCluster struct {
 	ID                 int                 `json:"id"`
 	Name               string              `json:"name"`
 	Domain             string              `json:"domain"`
+	Managed            bool                `json:"managed"`
```
Comment (svanharmelen) [inline instance_clusters.go:40]: @Starttoaster could you maybe check this one as well? I noticed the docs mention this as an option when adding a cluster (https://docs.gitlab.com/ee/api/instance_clusters.html#add-existing-instance-cluster) but not sure if it actually is a field in the response...

Comment (Starttoaster) [reply instance_clusters.go:40]: Absolutely! This looks like a self-hosted feature, and I'm personally a user of GitLab.com, but to be honest I've been meaning to deploy my own instance one of these days anyway. I'll take a look and get back to this.

Comment (svanharmelen) [reply instance_clusters.go:40]: Thanks! But I don't want to put a burden on your time, so please only have a look if it fits your personal schedule and interest!

**Pull request merged**